pkg/mount-manager/device-utils.go

```diff
@@ -214,27 +214,29 @@ func (m *deviceUtils) VerifyDevicePath(devicePaths []string, diskName string) (s
 }
 
 func udevadmTriggerForDiskIfExists(diskName string) error {
+	devToSCSI := map[string]string{}
 	sds, err := filepath.Glob(diskSDPattern)
 	if err != nil {
 		return fmt.Errorf("failed to filepath.Glob(\"%s\"): %v", diskSDPattern, err)
 	}
 	for _, devSDX := range sds {
 		scsiSerial, err := getScsiSerial(devSDX)
 		if err != nil {
-			return fmt.Errorf("failed to get SCSI Serial num for %s: %v", devSDX, err)
+			return fmt.Errorf("failed to get SCSI Serial num: %v", err)
 		}
+		devToSCSI[devSDX] = scsiSerial
 		if scsiSerial == diskName {
 			// Found the disk that we're looking for so run a trigger on it
 			// to resolve its /dev/by-id/ path
 			klog.Warningf("udevadm --trigger running to fix disk at path %s which has SCSI ID %s", devSDX, scsiSerial)
 			err := udevadmChangeToDrive(devSDX)
 			if err != nil {
-				return fmt.Errorf("failed to fix disk at path %s which has SCSI ID %s: %v", devSDX, scsiSerial, err)
+				return fmt.Errorf("failed to fix disk which has SCSI ID %s: %v", scsiSerial, err)
 			}
 			return nil
 		}
 	}
-	return fmt.Errorf("udevadm --trigger requested to fix disk %s but no such disk was found in %v", diskName, sds)
+	return fmt.Errorf("udevadm --trigger requested to fix disk %s but no such disk was found in %v", diskName, devToSCSI)
```

Comment:
Are errors like this going to bubble up to the user in k8s events? Do we actually want to give information to users about disks that may not be theirs?